composer.json

```
@@ -43,8 +45,10 @@
         "theofidry/alice-data-fixtures": "^1.5.2"
     },
     "conflict": {
-        "doctrine/annotations": "<1.2.7 || >=3.0",
-        "doctrine/dbal": "<2.11"
+        "doctrine/annotations": "<1.13.1 || >=3.0",
+        "doctrine/dbal": "<2.13.1 || ~3.0.0 || >=4.0",
```

```suggestion
        "doctrine/dbal": "<2.13.1 || ~3.0.0",
```

or

```suggestion
        "doctrine/dbal": "<2.13.1 || ~3.0.0 || >=5.0",
```

to not allow the next version that will be released?

---

`doctrine/dbal` 4.x is not stable yet, this is a good idea to not allow it yet. :+1: +1